pkg/api/v2/reporter/reporter.go

```diff
@@ -266,17 +269,17 @@ func (r *reporter) WriteReports(ctx context.Context, resourceErrs ResourceReport
 	return merr.ErrorOrNil()
 }
 
-// Ideally, this and its caller, WriteReports, would just take the resource ref and its status, rather than the resource itself,
-//    to avoid confusion about whether this may update the resource rather than just its status.
+// Ideally, this and its caller, WriteReports, would just take the resource ref and its status and/or messages, rather than the resource itself,
+//    to avoid confusion about whether this may update the resource rather than just its fields.
 //    However, this change is not worth the effort and risk right now. (Ariana, June 2020)
-func (r *reporter) attemptUpdateStatus(ctx context.Context, client ReporterResourceClient, resourceToWrite resources.InputResource, statusToWrite *core.Status) (resources.Resource, resources.InputResource, error) {
+func (r *reporter) attemptUpdate(ctx context.Context, client ReporterResourceClient, resourceToWrite resources.InputResource, statusToWrite *core.Status, messagesToWrite []string) (resources.Resource, resources.InputResource, error) {
```

Comment:
no longer need messagesToWrite? as messages is now a part of `core.Status`?

Reply:
Same as above